docs/note_dialogs.rst

```diff
@@ -49,12 +49,14 @@ And now it is also easy to mock ``AskNameAndAgeDialog.ask`` when testing the for
 .. code-block:: python
 
     def test_form_registration(qtbot, monkeypatch):
+        user = User.empty_user()
         form = RegistrationForm()
```

Comment:
```suggestion
        form = RegistrationForm(user)
```

Maybe? Seems difficult to imagine this particular empty `user` object magically changing behind the scenes somehow.